test/test_corpus.cc

```diff
@@ -605,120 +605,130 @@ TEST_P(LSPTest, All) {
     }
 
     // Quick fix code action
-    vector<shared_ptr<ApplyCodeActionAssertion>> applyCodeActionAssertions;
+    UnorderedMap<string, vector<shared_ptr<ApplyCodeActionAssertion>>> applyCodeActionAssertionsByFilename;
     for (auto &assertion : assertions) {
         if (auto applyCodeActionAssertion = dynamic_pointer_cast<ApplyCodeActionAssertion>(assertion)) {
-            applyCodeActionAssertions.push_back(applyCodeActionAssertion);
+            applyCodeActionAssertionsByFilename[applyCodeActionAssertion->filename].push_back(applyCodeActionAssertion);
         }
     }
+
     bool exhaustiveApplyCodeAction =
         BooleanPropertyAssertion::getValue("exhaustive-apply-code-action", assertions).value_or(false);
 
-    if (!applyCodeActionAssertions.empty() || exhaustiveApplyCodeAction) {
-        ASSERT_EQ(filenames.size(), 1) << "code-actions only works with tests that have a single file.";
+    if (!applyCodeActionAssertionsByFilename.empty() || exhaustiveApplyCodeAction) {
+        // ASSERT_EQ(filenames.size(), 1) << "code-actions only works with tests that have a single file.";
```

Comment:
why was this commented out?

Reply:
yes, it should be deleted since this change makes multi-file tests possible. thanks for the catch